docs/detections/api/rules/signals-api-overview.asciidoc

```diff
@@ -245,3 +250,107 @@ Example response:
   "failures": []
 }
 --------------------------------------------------
+
+
+==== Apply alert tags
+
+Add and remove alert tags.
+
+===== Request URL
+
+`POST <kibana host>:<port>/api/detection_engine/signals/tags`
+
+===== Request body
+
+A JSON object with the `tags` and `ids` fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`tags` |String[] a|Array of alert tags. Alert tags are the words and phrases that help categorize alerts. 
+
+Properties of the `tags` objects:
+
+* tags_to_add: (Required, string[]) Array of tags you want to add.
+* tags_to_remove: (Required, string[]) Array of tags you want to remove.
+
+NOTE: You cannot add and remove the same alert tag. 
+
+|No 
+
+|`ids` |String[] |Array of alert IDs.|Yes
+
+|==============================================
+
+====== Example requests
+
+Adds tags to alerts:
```

Comment:
Just noticed that this should probably say _Adds and removes alert tags_, since the example includes both actions.

Reply:
```suggestion
Adds and remove tags to alerts:
```